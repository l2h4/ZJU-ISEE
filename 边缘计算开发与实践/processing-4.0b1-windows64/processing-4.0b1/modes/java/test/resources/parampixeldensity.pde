void setup() {
  size(100, 100);
  // Pulling the display's density dynamically
  pixelDensity(displayDensity());
  noStroke();
}

void draw() {
  background(0);
  ellipse(30, 48, 36, 36);
  ellipse(70, 48, 36, 36);
}
