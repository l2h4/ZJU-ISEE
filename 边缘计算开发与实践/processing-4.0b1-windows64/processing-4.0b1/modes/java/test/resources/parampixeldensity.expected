import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class parampixeldensity extends PApplet {

  public void setup() {
    /* size commented out by preprocessor */;
    // Pulling the display's density dynamically
    /* pixelDensity commented out by preprocessor */;
    noStroke();
  }

  public void draw() {
    background(0);
    ellipse(30, 48, 36, 36);
    ellipse(70, 48, 36, 36);
  }

  public void settings() {
    size(100,100);
    pixelDensity(displayDensity());
  }

  static public void main(String[] passedArgs) {
      String[] appletArgs = new String[] { "parampixeldensity" };
      if (passedArgs != null) {
          PApplet.main(concat(appletArgs, passedArgs));
      } else {
          PApplet.main(appletArgs);
      }
  }

}
